infrastructure/Dockerfile

```diff
@@ -105,18 +106,18 @@ RUN wget https://github.com/hashicorp/consul-template/releases/download/v${CONSU
 # HAPROXY config
 #
 # Add files
-ADD haproxy.cfg        /etc/haproxy/haproxy.cfg
-# same file for an "empty template"
-ADD haproxy.cfg        /opt/consul-template/
-ADD template.conf      /opt/consul-template/
-ADD haproxy_reload.sh  /opt/consul-template/
-ADD consul-template.sh /opt/consul-template/
-ADD consul_config.json /etc/consul.d/
+RUN rm /etc/haproxy/haproxy.cfg                                 \
+    && ln -s /panteras/haproxy.cfg        /etc/haproxy/         \
+    && ln -s /panteras/haproxy.cfg        /opt/consul-template/ \
+    && ln -s /panteras/haproxy.conf.ctmpl /opt/consul-template/ \
+    && ln -s /panteras/haproxy_reload.sh  /opt/consul-template/ \
+    && ln -s /panteras/consul-template.sh /opt/consul-template/ \
+    && ln -s /panteras/consul_config.json /etc/consul.d/
```

Comment:
-1 

now `docker run panteras/paas-in-a-box` would not work
you have overcomplicated it now :)

- yes - for executing in one line most of commands 
- no - for mounting the stuff - otherwise you cannot run docker container and read the help